spec/models/ems_cluster_spec.rb

```diff
@@ -213,4 +213,30 @@
       expect(result).to be_falsey
     end
   end
+
+  context "#tenant_identity" do
+    # admin user is needed to process Events
```

Comment:
@lfu This comment does not apply anymore and should be removed.  Same with the other tests.